Sources/HipMobileUI/ViewModels/Pages/AppetizerViewModel.cs

```diff
@@ -26,7 +26,7 @@
 namespace PaderbornUniversity.SILab.Hip.Mobile.UI.ViewModels.Pages
 
 {
-    public class AppetizerPageViewModel : NavigationViewModel, IDownloadableListItemViewModel
+    public class AppetizerViewModel : NavigationViewModel, IDownloadableListItemViewModel
```

Comment:
Since this is not a *page* viewmodel anymore, it should be moved to the namespace `ViewModels.Views`, correct?
Or actually, isn't this still a page viewmodel? It is pushed on the navigation stack, so it's not just a view?

Reply:
You are right, It is still a page. Earlier Resharper was complaining while having Name in this way. I will fix this.